grid-compose/cmd/root.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-compose/internal"
+)
+
+var (
+	app        *internal.App
+	configPath string
```

Comment:
to avoid the global variable, we could add the app to the context. it will shared between the commands